test/controllers/orders_controller_test.rb

```diff
@@ -0,0 +1,57 @@
+require 'test_helper'
+
+class OrdersControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @order = orders(:one)
+  end
+
+  test "should get index" do
+    get orders_url :format => :json
```

Comment:
Should be able to:

```suggestion
    get orders_url, as: :json
```

as per [ActionDispatch::IntegrationTest](https://api.rubyonrails.org/v6.0.2.1/classes/ActionDispatch/IntegrationTest.html)